src/Samples/EventBindingsAndBehaviors/Program.fs

```diff
@@ -68,10 +69,16 @@ let bindings () : Binding<Model, Msg> list = [
 
 let designVm = ViewModel.designInstance (init ()) (bindings ())
 
-
 let main window =
-  Program.mkSimpleWpf init update bindings
-  |> Program.withConsoleTrace
-  |> Program.runWindowWithConfig
-     { ElmConfig.Default with LogConsole = true; Measure = true }
-     window
+
+  Log.Logger <- 
+    LoggerConfiguration()
+      .MinimumLevel.Override("Elmish.WPF.Update", Events.LogEventLevel.Verbose)
+      .MinimumLevel.Override("Elmish.WPF.Bindings", Events.LogEventLevel.Verbose)
+      .MinimumLevel.Override("Elmish.WPF.Performance", Events.LogEventLevel.Verbose)
+      .WriteTo.Console()
+      .CreateLogger()
+
+  WpfProgram.mkSimple init update bindings
+  |> WpfProgram.withLogger (new SerilogLoggerFactory())
```

Comment:
Instead of implicitly suggesting to users that they use Serilog's static logger, can we change all the samples to use a local logger.

```suggestion
  let logger =
    LoggerConfiguration()
      .MinimumLevel.Override("Elmish.WPF.Update", Events.LogEventLevel.Verbose)
      .MinimumLevel.Override("Elmish.WPF.Bindings", Events.LogEventLevel.Verbose)
      .MinimumLevel.Override("Elmish.WPF.Performance", Events.LogEventLevel.Verbose)
      .WriteTo.Console()
      .CreateLogger()

  WpfProgram.mkSimple init update bindings
  |> WpfProgram.withLogger (new SerilogLoggerFactory(logger))
```

Reply:
Is this based on some documented best practice for Serilog? Personally I always use the static logger. I find it a piece of global mutable state that simplifies rather than complicates. I have never found a benefit to explicitly passing loggers around as arguments; it's just noise IMHO. (If I was doing OO I'd pass them using DI, but I'm not, so I don't.)

Reply:
It is based on the principle of avoiding (static and) mutability.  I also always set the static logger, and I always log to it, but I don't always directly log to it.

For example, when I upgrade to Elmish.WPF 4.0 in my application at work, my logging will look more like this than the way the samples do now.
```F#
Log.Logger <-
  LoggerConfiguration()
    .WriteTo.Console()
    .CreateLogger()

let elmishWpfLogger =
  LoggerConfiguration()
    .MinimumLevel.Override("Elmish.WPF.Update", Events.LogEventLevel.Verbose)
    .MinimumLevel.Override("Elmish.WPF.Bindings", Events.LogEventLevel.Verbose)
    .MinimumLevel.Override("Elmish.WPF.Performance", Events.LogEventLevel.Verbose)
    .WriteTo.Logger(Log.Logger)
    .CreateLogger()

WpfProgram.mkSimple (fun () -> init) update bindings
|> WpfProgram.withLogger (new SerilogLoggerFactory(elmishWpfLogger))
|> WpfProgram.runWindow window
```

This code does a great job helping to demystify the magic category strings.

In contrast, consider the task of starting with the logging code in the sample and refactoring to what I have written above.  And suppose that code exists along side code that also passes Serilog loggers to Entity Framework Core and ASP.NET Core.  It is basically impossible to statically verify if the transformation is behavior preserving.  The names suggest they are, but just think about how many times you were completely confident that some change was behavior preserving but you ended up being wrong.  I know this has happened to me countless times.  I am getting better, but I am still not perfect.

Since I can't statically analyze the code, I next try to "statically analyze" the person that wrote it.  Did they not know that there was a simpler way to achieve the same behavior (and with the same amount of code)?  Did they know this but didn't do it out of laziness?  Or did they do it on purpose because there is some non-obvious reason that makes the approach better.  This is where a comment should be.  It is a comment that communicates "why".  Self-documenting code is a good thing, but it only communicates "how", not "why".

If you are not convinced though, we can keep code how it is.

Reply:
I'm not sure I follow all parts of your reasoning, but I don't really have a strong opinion here, so I made the change. :)